queries/flights.py

```diff
@@ -0,0 +1,502 @@
+"""
+
+    Greynir: Natural language processing for Icelandic
+
+    Flight schedule query response module
+
+    Copyright (C) 2021 Miðeind ehf.
+
+       This program is free software: you can redistribute it and/or modify
+       it under the terms of the GNU General Public License as published by
+       the Free Software Foundation, either version 3 of the License, or
+       (at your option) any later version.
+       This program is distributed in the hope that it will be useful,
+       but WITHOUT ANY WARRANTY; without even the implied warranty of
+       MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+       GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program. If not, see http://www.gnu.org/licenses/.
+
+
+    This module handles queries relating to air travel.
+
+"""
+
+import re
+import random
+import logging
+import cachetools
+from html import escape
+from typing import List, Dict, Any, Optional
```

Comment:
By convention, `from typing import...` should be the first import in the import list.

Reply:
Fixed!